lib/Controller/SettingsController.php

```diff
@@ -60,19 +60,20 @@ public function createProvisioning(array $data): JSONResponse {
 			$this->provisioningManager->newProvisioning($data);
 		} catch (ValidationException $e) {
 			return HttpJsonResponse::fail([$e->getFields()]);
+		} catch (\Exception $e) {
+			return HttpJsonResponse::fail([$e->getMessage()]);
 		}
 
 		return new JSONResponse([]);
 	}
 
 	public function updateProvisioning(int $id, array $data): JSONResponse {
 		try {
-			$this->provisioningManager->updateProvisioning(array_merge(
-				$data,
-				['id' => $id]
-			));
+			$this->provisioningManager->updateProvisioning(array_merge($data, ['id' => $id]));
 		} catch (ValidationException $e) {
 			return HttpJsonResponse::fail([$e->getFields()]);
+		} catch (\Exception $e) {
```

Comment:
same. Could this possibly be a ClientException? \Error will throw almost anything. That *anything* could also be from unexpected service errors, then the HTTP4xx isn't appropriate IMO

Reply:
`$this->provisioningManager->newProvisioning($data);` throws a validation exception if input data is invalid or a exception related to the database. I guess we can migrate this to Throwable and `HttpJsonResponse.errorFromThrowable`.